### DISCUSSION TITLE
"Minutes between events purges" - can someone help explain?

### DISCUSSION THREAD

**Discussion by comminutus** [evilsocket/opensnitch#986] [Q&A]
Hello fellow opensnitchers, can someone help me understand what the "Minutes between events purges" option is in the UI settings?  I understand the maximum number of days to keep events, just not sure what this setting does.

**Comment by gustavo-iniguez-goya** [comment]
Hi @comminutus ,

The option refers to how often we check the DB for old events. Depending on the amount of events, the DB can grow really quickly, so this option helps to keep it under control.
